src/components/posts/PostHeader.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import logo from "../../assets/TeachLogo.png";
+
+function PostHeader(props) {
+  return (
+    <div style={styles.containerStyles}>
+      <img src={logo} alt="" style={styles.imgStyles} />
```

Comment:
This shouldn't be an empty `alt` tag, it's poor accessibility. Add a brief description of what the image contains!